dot/core/service.go

```diff
@@ -222,7 +222,7 @@ func (s *Service) StorageRoot() (common.Hash, error) {
 
 func (s *Service) handleBlocks(ctx context.Context) {
 	for {
-		prev := s.blockState.BestBlockHash()
+		//prev := s.blockState.BestBlockHash()
```

Comment:
Are the comments intentional?

Reply:
yes this needs to be fixed and re-enabled. i will open an issue for it